ddsp/training/data.py

```diff
@@ -443,8 +447,15 @@ def features_dict(self):
         'note_active_frame_indices':
             tf.io.FixedLenFeature([self._feature_length * 128], tf.float32),
         'instrument_id': tf.io.FixedLenFeature([], tf.string),
+        'recording_id': tf.io.FixedLenFeature([], tf.string),
```

Comment:
Can you create a separate subclass? I believe the internal datasets don't have these keys and this will cause an error.

Reply:
Oh, actually this is from tfrecords under `gs://magentadata/datasets/urmp`. All tfrecords there have the keys I added. If we need to care about the compatibility of other URMP tfrecords, I can definitely make a separate subclass!

Reply:
Here is my external link of getting the tfrecords: https://console.cloud.google.com/storage/browser/magentadata/datasets/urmp

Reply:
okay, that's fine